src/main/java/org/generationcp/middleware/service/api/study/StudyService.java

```diff
@@ -44,7 +44,14 @@
 	 * @return StudyDetailsDto
 	 */
 	StudyDetailsDto getStudyDetailsForGeolocation(Integer geolocationId);
-	
+
+	/**
+	 *
+	 * @param geolocationId
+	 * @return StudyDetailsDto
+	 */
+	StudyDetailsDto getStudyDetailsByGeolocation(Integer geolocationId);
```

Comment:
I think it's confusing to have two methods that return the same object using the same parameter